### DISCUSSION TITLE
How to delete all faces/names?

### DISCUSSION THREAD

**Discussion by CWempe** [photoprism/photoprism#2512] [Q&A]
I set up a dedicated PhotoPrism instance for a little project (sharing wedding photos).
I imported some random photos for testing purposes.
Then I named some faces that were detected.

Now I want to delete these faces, because I do not want face recognition to be enabled any more.
I disabled it with `PHOTOPRISM_DISABLE_FACES="true"`(docker) now.
But it looks like I cannot undo the face recognition.
Is that correct?

I expected I could open the peolple menu and click "X" an each face (or multi select) to delete them.
But it looks like this is only for hiding faces (and requires premium status).

I think there should be an easy way to delete (all) face tags, clusters and names (without premium).

Am I missing something?
Is this already possible?
Should I create a feature request for this?

As a work around I can set up the instance from scratch.
But there might be bigger setups where this is not possible.

**Comment by CWempe** [comment]
I just found out I can untag and delete recognized faces for individual photos.
This might be the right way, but requires many many clicks per face.

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/updates/#facial-recognition

**Reply by CWempe** [reply]
Thanks. 👍 

How did I miss this?
Well, actually it is kind of hidden in the "Getting updates" section. 😉 

I think should be put/copied in another section like:
https://docs.photoprism.app/user-guide/organize/people/
https://docs.photoprism.app/user-guide/faq/
or https://docs.photoprism.app/getting-started/faq/

**Reply by lastzero** [reply]
Actually not, just was the fastest way to show it to you as I'm on vacation. Here is a link to all CLI examples: https://docs.photoprism.app/getting-started/docker-compose/#examples

You will also find this in the docker-compose.yml inline docs and of course when you simply run `photoprism help`.

**Comment by rickysarraf** [comment]
I think there is a 'faces reset' command line option that should do the
trick

s3nt fr0m a $martph0ne, excuse typ0s
